src/peep/peep.c

```diff
@@ -7131,70 +7131,115 @@ uint8 sub_69A60A(rct_peep* peep){
 
 /**
  *
+ *  rct2: 0x0069A997
+ */
+static int sub_69A997(sint16 x, sint16 y, uint8 z, int direction, rct_map_element *dest_map_element, int test_edge) {
+	int eax = x, ebx = direction, ecx = y, edx = z, esi = (int) dest_map_element, edi = 0xFFFF, ebp = test_edge;
+	RCT2_CALLFUNC_X(0x0069A997, &eax, &ebx, &ecx, &edx, &esi, &edi, &ebp);
+	return edi;
+}
+
+/**
+ *
  *  rct2: 0x0069A5F0
  */
-static int sub_69A5F0(sint16 x, sint16 y, sint16 z, rct_peep *peep, rct_map_element *map_element){
-	//RCT2_GLOBAL(0x00F1AEDC, uint8) = sub_69A60A(peep);
-
-	//// Redundant check to make sure peep is not null??
-	//sint16 start_x = RCT2_GLOBAL(0x00F1AECE, sint16);
-	//sint16 start_y = RCT2_GLOBAL(0x00F1AED0, sint16);
-	//uint8 start_z = RCT2_GLOBAL(0x00F1AED2, uint8);
-	//
-	//uint8 edges = 0xF;
-	//if (peep->var_CC.x == (start_x / 32) &&
-	//	peep->var_CC.y == (start_y / 32) &&
-	//	peep->var_CC.z == start_z){
-
-	//	uint8 index = 0;
-	//	for (; index < 4; ++index){
-	//		if (peep->var_D0[index].x == x &&
-	//			peep->var_D0[index].y == y &&
-	//			peep->var_D0[index].z == z){
-	//			edges = peep->var_D0[index].direction & 0xF;
-	//			break;
-	//		}
-	//	}
-	//}
+static int pathfind_choose_direction(sint16 x, sint16 y, sint16 z, rct_peep *peep, rct_map_element *map_element) {
```

Comment:
This function will only be used by guests so could you prepend `guest_` onto the function name.